ClientApp/app/auth/register.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState } from "react";
+import { View, Text, TextInput, Alert, TouchableOpacity } from "react-native";
+import { useForm, Controller } from "react-hook-form";
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import Checkbox from 'expo-checkbox';
+import { useRouter } from "expo-router";
+import ConfirmButton from "@/components/ConfirmButton";
+import AuthenticationDivider from "@/components/AuthenticationDivider";
+
+interface RegisterFormData {
```

Comment:
Do you want to keep the name and last name to the next part of the signup process and only keep username + password combination here ?

Reply:
I don't know what you mean by next part of the signup process. I think its better to keep it in the registration process so people can know who other people are in the long run. If a user only puts his username and never updates his first name later on in the app, it will be difficult for people to know who's who. I don't know if that answers your comment.